README.md

```diff
@@ -158,147 +130,137 @@ arbitrary but much be unique in this environment.
 
 ### Components vs Profiles
 
-If you are already organizing your code into roles and profiles the application
-components are probably very similiar to your profiles. If all your nodes serve
-a single purpose you may be able to just convert your profile classes into
-component defined types. If you need to put multiple components on a single
-node that share resources this may resut in conflicts.  For example if you have
-one database node that provide datbases for multiple wordpress application
-instances mysql puppet resources may be shared or if you had multiple http
-components in your stack apache resources might be shared. In this case the
-shared resources should be factored out of the components into profile classes
-which can then be included in the component. An example of this is
-`wordpress_app:database` and `workpress_app::database_profile`. The underlying
-mysql server and firewall rules are configured in the profile while the
-specific datbase, user, and permissions are managed in the component.
+If you're already organizing your code into *roles and profiles*, the application components are probably very similiar to your profiles. If all your nodes serve a single purpose, you may be able to just convert your profile classes into component defined types. However, if you need to put multiple components on a single node that share resources, this may result in conflicts. For example, if you have one database node that provides databases for multiple WordPress application instances, MySQL resources may be shared. Or, if you have multiple HTTP components in your stack, Apache resources might be shared. If this is the case, you should turn the components into profile classes that can then be included in the component. Consider `wordpress_app:database` and `workpress_app::database_profile`: the underlying MySQL server and firewall rules are configured in the profile while the specific datbase, user, and permissions are managed in the component.
```

Comment:
If you need to put multiple components that share resources on a single node?

The application is Wordpress or wordpress it is a puppet construct not WordPress(tm). MySQL HTTP and Apache are fine as they refer to those actual technologies.

datbase